argnorm/drug_categorization.py

```diff
@@ -2,8 +2,30 @@
 from . import lib
 
 ARO = lib.get_aro_ontology()
+
 confers_resistance_to_drug_class_rel = ARO.get_relationship('confers_resistance_to_drug_class')
 confers_resistance_to_antibiotic_rel = ARO.get_relationship('confers_resistance_to_antibiotic')
+has_part_rel = ARO.get_relationship('has_part')
+is_small_inhibitor_molecule_rel = ARO.get_relationship('is_small_molecule_inhibitor')
+
+def navigate_superclasses(super_classes_list: List[str]) -> List[str]:
```

Comment:
This is a bad name for this function: it sounds very generic, but does something very specialized for this usage.

If this function is not to be used by users, prefix its name with an underscore